client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -89,22 +88,14 @@ class PluginUpload extends Component {
 	};
 
 	renderUploadCard() {
-		const { inProgress, complete, isJetpack, isAtomic } = this.props;
+		const { inProgress, complete, isJetpack } = this.props;
 
 		const uploadAction = isJetpack
 			? this.props.uploadPlugin
 			: this.props.initiateAutomatedTransferWithPluginZip;
 
-		const WrapperComponent = ! isAtomic ? FeatureExample : Fragment;
```

Comment:
The only change here is dropping the `! isAtomic` test, since we can show this upload to non-atomic users now.

Reply:
I think we should add a check to verify they are on an eligible plan, otherwise don't allow upload